packages/runtime/container-runtime/src/opLifecycle/opGroupingManager.ts

```diff
@@ -22,6 +22,11 @@ interface IGroupedMessage {
 	compression?: string;
 }
 
+function isGroupContents(opContents: IGroupedBatchMessageContents | { type?: unknown } | undefined): opContents is IGroupedBatchMessageContents
+{
```

Comment:
I think prettier will not approve of this.  You'll want to run some command... maybe `pnpm prettier:fix` or something.  I have VS Code set up to format on save, something to consider enabling too so you don't have to think about it.

Reply:
PR checker should catch violations, right? I didn't even think about it because I expect that format on save is on for repo. I see now that some checks other than PR template complaint also failed. (Didn't see the scroll bar - that is a lot of checks :) )
`pnpm prettier:fix` fails in root for me, but VSCode plugin does work.